### PR TITLE
LPD-42244 Delete feature flag

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-12584)

### Changes

Delete the FF LPS-189856
Fix some wrong behaviours for actions
Adapt tests

## Test scenario 

1. Create some Display page Templates and Folders
2. Try to test every individual action for DPT and Folders and also try the managment toolbar ones.

 
[Screencast from 2024-11-20 13-39-06.webm](https://github.com/user-attachments/assets/bbcb5a8f-5a4f-44d1-b650-bfaf5760856f)

> [!NOTE]  
> The failing playwright tests seems not to be related to this 🤔

### CODE REVIEW

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorCustomView.java [hunk 1]
```diff
@@ -65,11 +64,7 @@ public boolean isVisible(
 		AssetDisplayPageSelectorCriterion assetDisplayPageSelectorCriterion,
 		ThemeDisplay themeDisplay) {
 
-		if (FeatureFlagManagerUtil.isEnabled("LPS-189856")) {
-			return true;
-		}
-
-		return false;
+		return true;
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorCustomView.java:67]: @BarbaraCabrera the default value is true, so you can remove this method

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorView.java [hunk 2]
```diff
@@ -61,11 +60,7 @@ public boolean isVisible(
 		AssetDisplayPageSelectorCriterion assetDisplayPageSelectorCriterion,
 		ThemeDisplay themeDisplay) {
 
-		if (FeatureFlagManagerUtil.isEnabled("LPS-189856")) {
-			return false;
-		}
-
-		return true;
+		return false;
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorView.java:63]: @BarbaraCabrera setting this to false means that it is not going to be used at all, so you can remove the class

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

**Comment by ealonso** [comment]
@BarbaraCabrera I think this commit [6dbfc61](https://github.com/liferay-page-management/liferay-portal/pull/16598/commits/6dbfc61834b3b19898cd00d4c246ab660169e26c) is not necessary, since we can create Display Pages in root folder.

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorView.java [hunk 3]
```diff
@@ -56,18 +54,6 @@ public String getTitle(Locale locale) {
 		return _language.get(locale, "display-page-templates");
 	}
 
-	@Override
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/AssetDisplayPagesItemSelectorView.java:59]: @BarbaraCabrera I think this class is not necessary and you need to remove it. We don't want to have two different views, we want to have only the one with folders

**Comment by ealonso** [comment]
@BarbaraCabrera all failures in displayPagesAdmin seems to be related with your changes

**Comment by BarbaraCabrera** [comment]
ci:test:page-management

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp [hunk 4]
```diff
@@ -64,35 +64,37 @@ AssetDisplayPagesItemSelectorCustomViewDisplayContext assetDisplayPagesItemSelec
...
 <aui:script sandbox="<%= true %>">
-	var selectItemHandler = Liferay.Util.delegate(
-		document.querySelector('#<portlet:namespace />displayPages'),
-		'click',
-		'.layout-page-template-entry',
-		(event) => {
-			var domElement = event.delegateTarget.closest('dd');
+	Liferay.componentReady('<portlet:namespace />displayPages' + '<%= assetDisplayPagesItemSelectorCustomViewDisplayContext.getLayoutPageTemplateCollectionId() %>).then(() => {
+		var selectItemHandler = Liferay.Util.delegate(
+			document.querySelector('#<portlet:namespace />displayPages'),
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp:69]: @BarbaraCabrera I think you also need to update this 'displayPages' id

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp [hunk 5]
```diff
@@ -64,35 +64,37 @@ AssetDisplayPagesItemSelectorCustomViewDisplayContext assetDisplayPagesItemSelec
 </clay:container-fluid>
 
 <aui:script sandbox="<%= true %>">
-	var selectItemHandler = Liferay.Util.delegate(
-		document.querySelector('#<portlet:namespace />displayPages'),
-		'click',
-		'.layout-page-template-entry',
-		(event) => {
-			var domElement = event.delegateTarget.closest('dd');
+	Liferay.componentReady('<portlet:namespace />displayPages' + '<%= assetDisplayPagesItemSelectorCustomViewDisplayContext.getLayoutPageTemplateCollectionId() %>).then(() => {
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp:67]: @BarbaraCabrera also here you can do 

```
<portlet:namespace />displayPages<%= assetDisplayPagesItemSelectorCustomViewDisplayContext.getLayoutPageTemplateCollectionId() %>
`
``
The plus button is not necessary

**Comment by BarbaraCabrera** [comment]
ci:stop

**Comment by BarbaraCabrera** [comment]
ci:test:page-management

**Comment by ealonso** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:stop

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp [hunk 6]
```diff
@@ -9,6 +9,7 @@
 
 <%
 AssetDisplayPagesItemSelectorCustomViewDisplayContext assetDisplayPagesItemSelectorCustomViewDisplayContext = (AssetDisplayPagesItemSelectorCustomViewDisplayContext)request.getAttribute(AssetDisplayPagesItemSelectorCustomViewDisplayContext.class.getName());
+AssetDisplayPagesItemSelectorCustomViewManagementToolbarDisplayContext assetDisplayPagesItemSelectorCustomViewManagementToolbarDisplayContext = (AssetDisplayPagesItemSelectorCustomViewManagementToolbarDisplayContext)request.getAttribute(AssetDisplayPagesItemSelectorCustomViewManagementToolbarDisplayContext.class.getName());
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp:12]: @BarbaraCabrera you need to add an empty line between

File: modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp [hunk 7]
```diff
@@ -64,35 +65,43 @@ AssetDisplayPagesItemSelectorCustomViewDisplayContext assetDisplayPagesItemSelec
...
 
 <aui:script sandbox="<%= true %>">
-	var selectItemHandler = Liferay.Util.delegate(
-		document.querySelector('#<portlet:namespace />displayPages'),
-		'click',
-		'.layout-page-template-entry',
-		(event) => {
-			var domElement = event.delegateTarget.closest('dd');
+	Liferay.componentReady(
+		'<portlet:namespace />' +
```
Comment (ealonso) [inline modules/apps/asset/asset-display-page-item-selector-web/src/main/resources/META-INF/resources/view.jsp:69]: @BarbaraCabrera ' + ' is not necessary you can just remove it.

**Comment by ealonso** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:test:page-management

**Comment by BarbaraCabrera** [comment]
ci:reevaluate:1407401_11979

**Comment by BarbaraCabrera** [comment]
ci:test:page-management

**Comment by ealonso** [comment]
ci:test:page-management

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**